libs/design/DetailCard/DetailCard.tsx

```diff
@@ -7,14 +8,32 @@ import { DetailCardTextField } from './DetailCardTextField';
 interface Props {
   title: string;
   id: string;
+  deleteButton: string;
   children: React.ReactNode;
+  onDelete: React.MouseEventHandler<HTMLButtonElement>;
 }
 
-export const DetailCard = ({ title, id, children }: Props) => {
+export const DetailCard = ({
+  title,
+  id,
+  children,
+  deleteButton,
```

Comment:
Should all "DetailCard"s have a delete button?

Reply:
If we want to have it for projects, probably yes. Because projects can also be deleted.

Reply:
This is an induction, and it's dangerous.
There is no rule that all DetailCards must have a delete button, so we should not include a delete feature as a concern.

Reply:
Okay got ur point.